### ISSUE TITLE
Test loss

### ISSUE THREAD

**Issue by raimis** [torchmd/torchmd-net#120]
Which function is supposed to compute the test loss? Can we clean it up?

https://github.com/torchmd/torchmd-net/blob/2f6e19091ce161edb86568fe9a2af291a69b38d3/torchmdnet/module.py#L61-L69

Ping: @PhilippThoelke @stefdoerr

**Comment by giadefa** [comment]
Because of the way lighting works we could only do it this way

**Comment by PhilippThoelke** [comment]
Loss is computed in `step`, no matter if during train, val or test. The function `validation_step` is called by pytorch-lightning to evaluate the model on all validation dataloaders. `test_step` is only called at the end of training or via `trainer.test(...)`. Since we want to compute test loss during training, one of the validation dataloaders contains the test set.
